CheckPrime.java

```diff
@@ -0,0 +1,40 @@
+import java.util.Scanner;
+
+public class CheckPrime {
+
+	public static void main(String[] args) {
+
+		int primeNum = primeFunction();
+
+		if (primeNum == 0) {
+
+			System.out.println(" It's a prime No");
+		} else {
+
+			System.out.println(" It's not a prime No");
+		}
+
+	}
+
+	public static int primeFunction() {
+
+		Scanner input = new Scanner(System.in);
```

Comment:
do these things in main function and and pass number in primeFunction as a parameter.